felix/bpf/tc/attach.go

```diff
@@ -481,13 +448,12 @@ func (ap *AttachPoint) ConfigureProgram(m *libbpf.Map) error {
 		globalData.JumpsV6[tcdefs.ProgIndexPolicy] = uint32(ap.PolicyIdxV6)
 	}
 
-	return ConfigureProgram(m, ap.Iface, &globalData)
+	ConfigureProgram(ap.Iface, globalData)
+	return globalData
 }
 
-func ConfigureProgram(m *libbpf.Map, iface string, globalData *libbpf.TcGlobalData) error {
+func ConfigureProgram(iface string, globalData *libbpf.TcGlobalData) {
```

Comment:
you can remove the `func ConfigureProgram` and do the little bit right in the ap